sphinx/directives/other.py

```diff
@@ -39,6 +39,7 @@ class TocTree(Directive):
     final_argument_whitespace = False
     option_spec = {
         'maxdepth': int,
+        'name': str,
```

Comment:
Another thought, IMO, Sphinx directives should accept `:name:` and `:class:` options to match to docutils directives.
`:name:` is used for node ID and `:class:` is used for node CLASS.
Two options can be applied for any docutils directives. but Sphinx directive doesn't follow it.
http://docutils.sourceforge.net/docs/ref/rst/directives.html#name